api/reports.py

```diff
@@ -234,36 +233,60 @@ def get_camera_with_most_violations():
     }
 
 
-@reports_router.get("/{camera_id}/daily_data")
-def get_daily_data(camera_id: str, date: date = Query(date.today().isoformat())):
+@reports_router.get("/camera/{camera_id}/daily_data")
```

Comment:
I think that in the future we will have 3 namespaces in the reports API.

- `reports/processor/`: Contains reports with the processor information (for example memory usage)
- `reports/cameras/`: Contains camera reports (for example, social distancing metrics)
- `reports/processor/`: Contains area reports (for example, social distancing metrics)

I didn't change that in the context of this PR to avoid changing the current frontend.
Whit the dashboard redesign we can change the structure of the reports namespace.

Reply:
I agree!